api/src/main/resources/AlignCaseManagerCaseRelationships.sql

```diff
@@ -0,0 +1,30 @@
+
+SET FOREIGN_KEY_CHECKS=0;
+
+UPDATE relationship_type SET b_is_to_a = 'Case' WHERE uuid = '9065e3c6-b2f5-4f99-9cbf-f67fd9f82ec5';
+
+UPDATE relationship r
```

Comment:
Do we want to limit this to the entries created using 3.x? I know we already had this relationship before but am not sure how this was implemented in 2.x

Reply:
The second query below that one does the 2x migrations - looks like this: 
UPDATE relationship r
	INNER JOIN relationship_type t ON r.relationship = t.relationship_type_id
	SET r.person_a = r.person_b , r.person_b = r.person_a
    WHERE  t.uuid = '9065e3c6-b2f5-4f99-9cbf-f67fd9f82ec5';